app/controllers/concerns/course/forum_topics_concern.rb

```diff
@@ -0,0 +1,35 @@
+module Course::ForumTopicsConcern
```

Comment:
A concern is a logical group of functionality. This group of functionality doesn't seem to have any logical connection to each other. Split it up.